raiden/tests/integration/long_running/test_settlement.py

```diff
@@ -322,7 +321,7 @@ def run_test_batch_unlock(
         secret=secret,
     )
 
-    gevent.sleep(1)  # wait for the messages to be exchanged
+    secret_request_event.get()  # wait for the messages to be exchanged
```

Comment:
I promised i would do this in a separate PR a while ago, thanks for doing these changes